src/docs/background/23-how-capabilities-work/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How Capabilities work?
```

Comment:
This article seems to be discussing the changes and migration, rather than just providing background on the capabilities framework, so I think "Changes to Stripe provider onboarding and Stripe capabilities framework" would be better.